doc/fwutil/fwutil.md

```diff
@@ -158,20 +173,21 @@ In order to improve scalability and performance a modern network switches provid
 1. Non modular chassis platforms
 2. Modular chassis platforms
 
-Non modular chassis platforms may contain only one chassis.  
```

Comment:
@sujinmkang why do we need to remove these whitespaces? Basically in .md two space notation is used to start a new line which allows text separation.